### ISSUE TITLE
Suggestion: Upload Architecture Specific APK

### ISSUE THREAD

**Issue by VedxP** [ImranR98/Obtainium#91]
Currently, the app installation size is more than 45 MB.

Releasing separate APK for separate architecture would make the installation size much smaller.

# :)

**Comment by ImranR98** [comment]
Good idea, I've added split APKs to the latest release and will include them moving forward.

**Comment by VedxP** [comment]
> Good idea, I've added split APKs to the latest release and will include them moving forward.

Voila :)
